includes/class-wc-stripe.php

```
@@ -252,8 +251,8 @@ public function init() {
 		$intent_controller = new WC_Stripe_Intent_Controller();
 		$intent_controller->init_hooks();
 
-		$checkout_sessions_controller = new WC_Stripe_Checkout_Sessions_Controller();
-		$checkout_sessions_controller->init_hooks();
+		$checkout_sessions_ajax_handler = new \WC_Stripe\Ajax_Handlers\WC_Stripe_Checkout_Sessions_Ajax_Handler();
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '254', column '41'). (undefined)

(MissingImport)